### ISSUE TITLE
Can't click option in context menu (right click menu) on mobile device

### ISSUE THREAD

**Issue by bl4ckswordsman** [Lissy93/dashy#1162]
### Environment

Self-Hosted (Docker)

### System

Android 12, Chrome 112

### Version

2.1.1

### Describe the problem

On my phone, when I touch and hold an icon in Dashy to show its context menu (right click menu on desktop) list, I can only click the list elements the first time I open the menu. After the first time, the list disappears before the click/touch is registered. I have to reload the page to be able to click an element in the menu again. Sometimes, this issue (not being able to click an element in the context menu) happens even after I scroll in the page.

### Additional info

https://user-images.githubusercontent.com/91825788/231539281-5d7485d0-97a8-473d-8383-a8e40f8bee2a.mp4

### Please tick the boxes

- [X] You have explained the issue clearly, and included all relevant info
- [X] You are using a [supported](https://github.com/Lissy93/dashy/blob/master/.github/SECURITY.md#supported-versions) version of Dashy
- [X] You've checked that this [issue hasn't already been raised](https://github.com/Lissy93/dashy/issues?q=is%3Aissue)
- [X] You've checked the [docs](https://github.com/Lissy93/dashy/tree/master/docs#readme) and [troubleshooting](https://github.com/Lissy93/dashy/blob/master/docs/troubleshooting.md#troubleshooting) guide ![important](https://img.shields.io/badge/Important!-F6094E)
- [X] You agree to the [code of conduct](https://github.com/Lissy93/dashy/blob/master/.github/CODE_OF_CONDUCT.md#contributor-covenant-code-of-conduct)

**Comment by bl4ckswordsman** [comment]
Still relevant.

**Comment by bl4ckswordsman** [comment]
Still relevant

**Comment by ttomasini** [comment]
push

**Comment by CrazyWolf13** [comment]
Can't really recreate this issue, can you confirm it is still a problem?
For me this works just fine.

What browser are you using?
Which phone brand? Google Pixel?

**Comment by bl4ckswordsman** [comment]
> Can't really recreate this issue, can you confirm it is still a problem? For me this works just fine.
> 
> What browser are you using? Which phone brand? Google Pixel?

I opened this issue almost a year ago, at that time I used it in Chrome webview with a OnePlus 7 Pro. Now I'm still on Chrome but on a Pixel 8 Pro.

I think the issue I've described is way less frequent now than what it was back then. Now it occasionally occurs when I repeat the steps of single clicking an entry, going back a few times, and then retrying the "press and hold".

But this issue can be closed now perhaps.
